api-reference/_hidden/ExcelExportBaseProps/encodeExecutableContent.md

```diff
@@ -5,7 +5,9 @@ default: false
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies whether to encode potentially dangerous content when it is exported to a CSV file.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+Exported data can contain executable content such as formulas. This content makes it possible to execute malicious commands if a user opens a file in the spreadsheet application and confirms that the file can be loaded and commands executed.
+
+Enable this property to encode formulas into simple data when the content is exported to CSV.
```

Comment:
Exported where? To a file.

Exported how? In CSV format.

Besides, we've already explained everything above, so there is no need to repeat things over and over.

```suggestion
Enable this property to ensure that exported CSV files are safe for loading in third-party spreadsheet applications.
```

P.S. I wasn't aware of the fact that CSV files can include executable content. CSV is just "Comma-Separated Values" by its definition. One of the simplest plain text formats ever. Well, live and learn, I guess.